NitroxModel/Platforms/Store/Steam.cs

```diff
@@ -16,10 +17,11 @@ public sealed class Steam : IGamePlatform
         public static Steam Instance => instance ??= new Steam();
 
         public string Name => nameof(Steam);
+        public Platform platform => Platform.STEAM;
 
         public bool OwnsGame(string gameDirectory)
         {
-            return File.Exists(Path.Combine(gameDirectory, "steam_api64.dll"));
+            return File.Exists(Path.Combine(gameDirectory, "steam_api64.dll")) || File.Exists(Path.Combine(gameDirectory, "Subnautica_Data", "Plugins", "CSteamworks.dll"));
```

Comment:
This extra check is game specific, not platform specific code. If it's not necessary I prefer to remove it or change it to something that every steam game has.